### DISCUSSION TITLE
slack community link

### DISCUSSION THREAD

**Discussion by pyther-hub** [JdeRobot/jderobot.github.io#238] [General]
Hi all, My name is Sudhanshu Rai I am a sophomore at IIT ISM, I found about open source community through gsoc and I would love to contribute to the open source project.
I have advanced knowledge of python and have great knowledge in machine learning.
someone please provide me the slack community link
@jmplaza

**Comment by jmplaza** [comment]
Hi @pyther-hub, 

welcome to our community. We mainly use Slack only for consolidated contributors. A good way to start contributing is to get familiar with our JdeRobot software products. In particular, RoboticsAcademy is the most active one and a priority project. The best way to start contributing is to USE RoboticsAcademy (solve and provide feedback) about the exercises already available in [current release](https://jderobot.github.io/RoboticsAcademy/exercises/) (FollowLine, ObstacleAvoidance, VacuumCleaner, LocalizedVacuumCleaner...). They use the browser as the only GUI (for source code editing and for exercise monitoring), and a docker image for running the Gazebo simulator (it is named RADI= RoboticsAcademy Docker Image). The docker image already includes a Django webserver for providing the exercise webpages. Please use this GitHub Discussions for questions (you will receive support here) and report any detected bug creating an issue for each one in the [RoboticsAcademy repository](https://github.com/JdeRobot/RoboticsAcademy). Feel free to share videos in the forum or mentioning @JdeRobot in twitter of your solutions to those exercises in operation. It also includes some exercises on DeepLearning: digit classifier and human detection.

After using RoboticsAcademy, a second step in contributing is studying its source code and improve it. Other active JdeRobot projects are [VisualCircuit](https://github.com/JdeRobot/VisualCircuit) ([illustrative video](https://www.youtube.com/watch?v=WMRDqv3TYB0)) and [FPGA-Robotics](https://github.com/JdeRobot/FPGA-robotics) ([illustrative video](https://www.youtube.com/watch?v=rbdQ36ZJ7Lo)). We have one Slack channel per project and several WorkingGroups for solid contributors.

In addition, in JdeRobot community we have a Robotics Club, on it we help students to improve their robotics abilities through periodic meetings with JdeRobot mentors, experts on robotics, and collaboration on some specific projects. Each member of the Club has his/her own repository and blog, beyond working on official code repositories too. Take a look at [Sakshay's blog](https://theroboticsclub.github.io/colab-Sakshay_Mahna). The main advantage on your side of joining the Robotics Club is to learn more robotics abilities, contribute to nice open source projects and to receive mentoring from robotics experts, for free. After doing a good betatesting of RoboticsAcademy you may be invited to the RoboticsClub, if interested. This club is independent of Google Summer of Code activities, where we follow a different and merit based candidate selection process.

Cheers!

**Comment by Arunjagan12** [comment]
Greetings to all. I'm Arun, a second-year computer science student and aspiring programmer. After working on a few github projects, I discovered open source projects. I want to find out more about how I can help with these projects. One of your initiatives that I came upon caught my attention. I want to contribute more to this initiative and grow as a person. I know more about HTML, CSS, and javascript, and I'm now learning about react. I also completed a web development internship, which enabled me to work on projects professionally.
Could someone please give me the link to the Slack community?
@jmplaza

**Reply by jmplaza** [reply]
Hi @Arunjagan12 , 

please take a look at the previous response in this thread.

We mainly use Slack only for consolidated contributors, for the general community we use GitHub Discussions and issues at the project repos. For instance [RoboticsAcademy repo](https://github.com/JdeRobot/RoboticsAcademy), [VisualCircuit repo](https://github.com/JdeRobot/visualcircuit) or [FPGA-Robotics repo](https://github.com/JdeRobot/FPGA-robotics).

The best way to start contributing to JdeRobot is to USE RoboticsAcademy, solve and provide feedback about the available exercises. After doing a good betatesting of RoboticsAcademy and some contributions you will be invited to the Slack, if interested.

**Comment by GauravKumar9920** [comment]
just a basic doubt, what ubuntu version is reccomended for robotics academy, I am installing the devolper version on 18.04.

**Reply by jmplaza** [reply]
All robotics dependencies are pre-installed in the RADI (RoboticsAcademy Docker Image), which is linux based. The RoboticsAcademy user may have a Windows machine, a Linux machine or a MacOS machine, the only requirement is to support docker containers. 

As a RoboticsAcademy developer it is convenient to use a Linux machine, for instance Ubuntu Linux 20.04, to have the same environment that inside RADI (ROS1 Noetic, Gazebo 11, etc.)

**Comment by GauravKumar9920** [comment]
Hey @jmplaza @pariaspe I have actually sent a merge request for issue #1900  i.e remove world duplicates, can you pls take a look at it and provide with a response :)

**Comment by NAF1S** [comment]
Hello everyone,I am lokman sharif from Bangladesh. I am currently pursuing my undergrad at University of Dhaka.I know python(django),javascript(React) well.I want contribute to this community. Can I have slack group link of this community. 

Thanks in advance.

**Reply by jmplaza** [reply]
Hi @NAFI5O , 

first of all read the documentation :-) Including the [answer to that question](https://github.com/JdeRobot/RoboticsAcademy/discussions/1894#discussioncomment-4455796), just in this thread. For particular questions feel free to create a corresponding issue or item in Discussions. Some community member will help you _whenever possible_.

**Comment by GauravKumar9920** [comment]
Hey, I am really need help setting up the RADI locally. I am new to docker and I am not able to test the changes I made locally on my device. I went through the documentation but found it difficult to implement it. I successfully was able to follow steps of "Edit code on RADI On The GO." but the next one "How to add your local changes to RADI while persisting changes two-way" I'm not able to follow this correctly maybe evrerytime I run the command "docker run --rm -it -p 8000:8000 -p 2303:2303 -p 1905:1905 -p 8765:8765 -p 6080:6080 -p 1108:1108 -v $(pwd):/home jderobot/robotics-academy" the server opens the orignal docker image and not the one I locally made changes on. 

pls help me with this I tried a lot and I am stuck, It would be good if we can have a google meet or something if possible, I am actilvely looking forward to contribute to this org, thankyou!

**Reply by pariaspe** [reply]
Check my [answer](https://github.com/JdeRobot/RoboticsAcademy/issues/1910#issuecomment-1416190660) to your question :)
PS: Please next time open a new discussion with your question, since your question is not related with the original topic

**Comment by GauravKumar9920** [comment]
Are there any issues related to ROS or the ones which haven't been assigned or solved yet.

**Reply by pariaspe** [reply]
I have one in mind related to Gazebo, I'll ping you :+1: 
PS: Again, new discussion...
